pkg/kubectl/cmd/util/helpers.go

```diff
@@ -45,7 +46,8 @@ import (
 )
 
 const (
-	ApplyAnnotationsFlag = "save-config"
+	ApplyAnnotationsFlag      = "save-config"
+	ChangeSourceAnnotationKey = "kubectl.kubernetes.io/change-source"
```

Comment:
@janetkuo Please change to kubernetes.io/change-cause

@ghodss We'll add support for adding opaque annotations and then just copy all annotations from the Deployment to the RCs.